File: C/Exercicio30.c
#include<stdio.h>

int main(int argc, char *argv[]){

  int num1=1, num2=1, i, soma;
  
  printf("\nOs primeiros 30 numeros da sequencia de Fibonacci sao: ");
  printf("\n%i %i",num1,num2);

  for(i=2; i<30; ++i){
    soma=num1+num2;    
    printf(" %d",soma);    
    num1=num2;    
    num2=soma;    
  }  
  return 0;  
}

File: Python/Exercicio30.py
num1=1
num2=1

print("Os primeiros 30 numeros da sequencia de Fibonacci sao: ")
print("", num1, "\n", num2)

for i in range (2, 30, 1):
    soma=num1+num2 
    print("", soma)
    num1=num2
    num2=soma